src/magic_search.js

```diff
@@ -400,9 +401,10 @@ angular.module('MagicSearch')
                 // showMenu and hideMenu depend on foundation's dropdown. They need
                 // to be modified to work with another dropdown implemenation (i.e. bootstrap)
                 $scope.showMenu = function() {
+                    var facetDrop = $element.find('#facet-drop');
```

Comment:
I can see that this would pin down the #facet-drop acted on in the $timeout. But, how does this affect the bootstrap version you use?